packages/gatsby-plugin-manifest/src/gatsby-ssr.js

```diff
@@ -31,16 +31,23 @@ exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
       href={withPrefix(`/manifest.webmanifest`)}
     />
   )
-
   // The user has an option to opt out of the theme_color meta tag being inserted into the head.
   if (pluginOptions.theme_color) {
-    headComponents.push(
-      <meta
-        key={`gatsby-plugin-manifest-meta`}
-        name="theme-color"
-        content={pluginOptions.theme_color}
-      />
+    let theme_color = !Object.keys(pluginOptions).includes(
+      `theme_color_in_head`
     )
```

Comment:
this can be:
```
const insertMetaTag = `theme_color_in_head` in pluginOptions 
  ? pluginOptions.theme_color_in_head
  : true
```
We already know that `pluginOptions.theme_color` is truthy, because we are in `if (pluginOptions.theme_color) {`